### PR TITLE
WFLY-14263 Updating the outdated 'Remote EJB invocations via JNDI - E…

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-14263

### CODE REVIEW

**Comment by bstansberry** [comment]
@chengfang or @tadamski please review

File: docs/src/main/asciidoc/_developer-guide/Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project.adoc [hunk 1]
```diff
@@ -1,59 +1,33 @@
-[[Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_remote-naming_project]]
-= Remote Jakarta Enterprise Beans invocations via JNDI - Jakarta Enterprise Beans client API or remote-naming project
+[[Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project]]
+= Remote Jakarta Enterprise Beans invocations via JNDI - Jakarta Enterprise Beans client API or wildfly-naming-client project
 
 [[purpose]]
 == Purpose
 
-WildFly provides EJB client API project as well as remote-naming project
+WildFly provides EJB client API project as well as wildfly-naming project(https://github.com/wildfly/wildfly-naming-client)
```
Comment (tadamski) [inline docs/src/main/asciidoc/_developer-guide/Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project.adoc:7]: small issue but it should be: wildfly-naming-client project, there is also a number of such typos below; I believe those should be changed

Comment (RanabirChakraborty) [reply docs/src/main/asciidoc/_developer-guide/Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project.adoc:7]: Thanks for the update @tadamski, I'll change it accordingly.

Comment (RanabirChakraborty) [reply docs/src/main/asciidoc/_developer-guide/Remote_Jakarta_Enterprise_Beans_invocations_via_JNDI_-_Jakarta_Enterprise_Beans_client_API_or_wildfly-naming-client_project.adoc]: Done @tadamski

**Review by tadamski — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks, @RanabirChakraborty

**Pull request merged**